src/Learning/KWDataUtils/KWDatabaseIndexer.cpp

```diff
@@ -861,44 +861,44 @@ boolean KWDatabaseIndexer::ComputeRootTableIndexation()
 			if (lMaxKeyNumber < 0)
 				lMaxKeyNumber = 0;
 
-			// On limite ce nombre de cle on fonction de la taille des fichiers a analyser
+			// On limite ce nombre de cle on fonctioRate = 0; de la taille des fichiers a analyser
```

Comment:
I think this comment should be rewritten (replace "Rate = 0;" with "n" ?)

Reply:
Corrigé